client.go

```diff
@@ -253,6 +254,7 @@ func (cc *clientConn) RoundTrip(ctx context.Context, req *protocol.Request, rsp
 
 	handleResponseHeaders := func() error {
 		res := cs.res
+		res.Header.SetProtocol(consts.HTTP20)
```

Comment:
HTTP2？

Reply:
In `Request`, it sets `consts.HTTP20` as protocol too. I think here should follow the http version described in the rfc documentation